src/banner/Banner.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+
+import { QrCard, QrCardProps } from '../qrCard'
+import { BannerElements, StyledBanner } from './Banner.style'
+
+type Copy = Readonly<{ heading: string; paragraph: string }>
+
+type CTALinks = Array<JSX.Element>
+type CTAButtons = Array<JSX.Element>
+
+export type BannerProps = Readonly<{
+  copy: Copy
+  actions?: CTALinks | CTAButtons
+  qrcode?: QrCardProps
+}>
+
+export const Banner = ({ copy, actions, qrcode }: BannerProps) => (
+  <StyledBanner>
+    <BannerElements.Content>
+      <BannerElements.Body>
+        <BannerElements.Heading as="h1">{copy.heading}</BannerElements.Heading>
+        <BannerElements.Paragraph as="p">{copy.paragraph}</BannerElements.Paragraph>
+        {actions && <BannerElements.Actions>{actions}</BannerElements.Actions>}
+      </BannerElements.Body>
+      <BannerElements.Media>
+        {qrcode && (
+          <BannerElements.Frame>
+            <QrCard imageUrl={qrcode.imageUrl} mainTitle={qrcode.mainTitle} />
```

Comment:
It seems the `Card` component used by `QrCard` only outputs a `li`, it might need refactoring if we want to use it as a standalone card with a `div`.

Reply:
Yes.. the DOM isn't what we are looking for here. Mainly the visual part, but I would not include that on our MVP.